server.js

```diff
@@ -129,7 +129,7 @@ function initialiseTemplateEngine (app) {
 }
 
 function initialisePublic (app) {
-  app.use('/.well-known/apple-developer-merchantid-domain-association.txt', express.static(path.join(__dirname, `/app/assets/apple-pay/${process.env.ENVIRONMENT}/apple-developer-merchantid-domain-association.txt`)))
+  app.use('/.well-known/apple-developer-merchantid-domain-association.txt', express.static(path.join(__dirname, `/app/assets/apple-pay/${process.env.ENVIRONMENT}/.well-known/apple-developer-merchantid-domain-association.txt`)))
```

Comment:
If we do this, we'll need to move [the 3 files for test-12, staging-2 and production-2](https://github.com/search?q=repo%3Aalphagov%2Fpay-frontend%20apple-developer&type=code)?